features/org.eclipse.equinox.core.sdk/feature.xml

```diff
@@ -74,13 +74,11 @@
    <plugin
          id="org.eclipse.equinox.security.win32"
          os="win32"
-         arch="x86_64"
```

Comment:
Good catch. I forgot that in https://github.com/eclipse-equinox/equinox/pull/564.
Can you please apply these two changes in this file in a separate PR mentioning it was forgotten?

Reply:
I thought you intentionally left them out of #564 ! because it was quite logical to me that you left them out, for this PR to fix them for WoA :) But now I see it was a misunderstanding on my part, please create a new PR for them and I'll remove them from this PR.

Reply:
Done via https://github.com/eclipse-equinox/equinox/pull/573.



> I thought you intentionally left them out of #564 ! because it was quite logical to me that you left them out, for this PR to fix them for WoA :)

That is a somewhat philosophical question since, at the moment, there is only the x86_64 arch supported on Windows so there is practically not a real difference. But since the plugin could/should run on all archs I think it is more suitable to remove it now.